README.md

```diff
@@ -31,6 +31,100 @@ insight.getUtxos('1Bitcoin...', function(err, utxos) {
 });
 ```
 
+You can optionally pass a minimum confirmation amount, and getUtxos will only return unspent transactions with at least that many confirmations.
+
+```javascript
+var explorers = require('bitcore-explorers');
```

Comment:
Shouldn't explorers import `litecore-explorers`? Also please repeat for all import examples below.